include/mbedtls/block_cipher.h

```diff
@@ -53,7 +53,6 @@ typedef struct {
     mbedtls_block_cipher_id_t MBEDTLS_PRIVATE(id);
 #if defined(MBEDTLS_BLOCK_CIPHER_SOME_PSA)
     mbedtls_block_cipher_engine_t engine;
-    psa_cipher_operation_t psa_operation;
     psa_key_type_t psa_key_type;
```

Comment:
Note: this seems somewhat redundant with `mbedtls_block_cipher_id_t` above. Could we avoid storing this one?